tests/foreman/api/test_provisioning.py

```diff
@@ -121,6 +121,10 @@ def test_rhel_pxe_provisioning(
         provisioning_host.os_version == expected_rhel_version
     ), 'Different than the expected OS version was installed'
 
+    # Verify provisioning log exists on host at correct path
+    assert provisioning_host.execute('test -f /root/install.post.log').status == 0
```

Comment:
Should we check for some content in this file? What if it's created but stays empty?

Reply:
sure, good point, `s/-f/-s` should do that

Reply:
@rplevka Good idea, will open up a separate PR to make it available in artifacts